translations/README.italian.md

```diff
@@ -0,0 +1,231 @@
+# Hasura GraphQL Engine
+
+[![Latest release](https://img.shields.io/github/v/release/hasura/graphql-engine)](https://github.com/hasura/graphql-engine/releases/latest)
+[![Docs](https://img.shields.io/badge/docs-v2.x-brightgreen.svg?style=flat)](https://hasura.io/docs)
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://hasura.io/newsletter/"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura è un prodotto open source che accelera lo sviluppo API di 10 volte fornendoti all'istante [GraphQL](https://hasura.io/graphql/) o API REST con autorizzazione integrata sui tuoi dati.
+
+Leggi di più su [hasura.io](https://hasura.io) e nei [docs](https://hasura.io/docs/).
+
+------------------
+
+![Hasura GraphQL Engine Demo](../assets/demo.gif)
+
+------------------
+
+![Hasura GraphQL Engine Realtime Demo](../assets/realtime.gif)
+
+-------------------
+
+## Features
+
+* **Crea queries potenti**: Filtro, paginazione, ricerca pattern, inserimento in bulk, aggiornamento ed eliminazione mutazioni integrati
```

Comment:
**rephrase**: I would use

`Filtri, paginazione, ricerca di pattern, e mutazioni di inserimento, aggiornamento ed eliminazione, anche in bulk`